src/gui/QvisStripChart.C

```diff
@@ -565,20 +565,77 @@ void QvisStripChart::addDataPoint( const unsigned int index,
 
     // Set the new Y value.
     vars[index].varData[0] = y;
-    
+
     // Update the samples for all curves.
     updateSamples();
-    
+
     updateAxis();
-    
+
+    replot();
+}
+
+//****************************************************************************
+// Class: QvisStripChart::addDataPoint
+//
+// Purpose: Add the next point to a curve.
+//
```

Comment:
Remove extra empty comment line. Move purpose comment to next line.

Reply:
fixed there an else where